yb-voyager/cmd/importData.go

```diff
@@ -858,10 +858,11 @@ func cleanImportState(state *ImportDataState, tasks []*ImportFileTask) {
 		nonEmptyTableNames := lo.Map(nonEmptyNts, func(nt sqlname.NameTuple, _ int) string {
 			return nt.ForOutput()
 		})
-		utils.PrintAndLog("Following tables are not empty. "+
-			"TRUNCATE them before importing data with --start-clean.\n%s",
-			strings.Join(nonEmptyTableNames, ", "))
-		yes := utils.AskPrompt("Do you want to continue without truncating these tables?")
+		utils.PrintAndLog("Empty tables: [%s]", strings.Join(nonEmptyTableNames, ", "))
+		utils.PrintAndLog("The above list of tables on target DB are not empty; " +
+			"partial data may have been imported in a previous run. Are you sure you want to start-clean? \n" +
```

Comment:
`partial data may have been imported in a previous run.` This statement seems a bit odd from `import-data-file` command

Reply:
Why? `import-data-file` is also resumable, so it's possible for a previous run to have imported partial data.

Reply:
Yeah, partial data might be confusing here.
By partial what does it mean
Q. Subset of tables is imported i.e. data is partially imported
Q. Set of tables is partially imported...

Do we really to make it verbose?

Reply:
```
utils.PrintAndLog("The above list of tables on target DB are not empty. Are you sure you want to start-clean? \n"
```

Reply:
I guess the intention behind "partial data"  is to explain why these tables might have been non-empty.. 
but yeah, agreed - "partial" is not clear. Them being non-empty already conveys that there is some data already imported.

Reply:
I was thinking of the scenario where the user has run the import-data-file for the first time with start-clean and has some data there already but still wants to load a file, for them this may sound like odd msg. 🤔